protos/telemetry/telemetry.proto

```diff
@@ -286,3 +295,47 @@ enum LandedState{
     LANDED_STATE_TAKING_OFF = 3;
     LANDED_STATE_LANDING = 4;
 }
+
+// Odometry message type.
+message Odometry {
+    // Mavlink frame id
+    enum MavFrame {
```

Comment:
Sorry, I keep on being annoying here. Should we call it `Frame` instead of `MavFrame`?

Reply:
Hm. I think that `Frame` can be little confusing because of `MavFrame` (`MAV_FRAME`) is established term in docs and code

Reply:
Ok.